.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com/
+
+[o:openmrs:p:OpenMRS:r:reporting-ui-module]
```

Comment:
This doesn't look right.  Shouldn't this be updated in the reportingui module, not in the reporting module?

Reply:
okay let me fix the end poiint ASAP